inst/pandoc_templates/jolla-blue.html

```diff
@@ -12,6 +12,21 @@
     <style type="text/css">body {background-color: #3A66B7;font-family: 'Merriweather', serif;color: #FEFEFA;}</style>
 
     <title>$title$</title>
+    
+    $if(meta)$
+    <meta name="description" content="$meta.description$"></meta>
+      <meta name="twitter:description" content="$meta.description$"></meta>
+      <meta name="og:description" content="$meta.description$"></meta>
+      <meta name="twitter:title" content="$title$"></meta>
+      <meta name="og:title" content="$title$"></meta>
+      <meta name="twitter:url" content="$meta.url$"></meta>
+      <meta property="og:url" content="$meta.url$"></meta>
+      <meta name="twitter:image:src" content="$meta.url$/$image$"></meta>
+      <meta property="og:image" content="$meta.url$/$image$"></meta>
+      <meta name="twitter:card" content="summary"></meta>
+      <meta name="twitter:creator" content="@$meta.twitter$"></meta>
```

Comment:
I think you should make these conditional so they only show up if the twitter field is supplied